.github/workflows/fuzz-cmd.yml

```
@@ -25,7 +25,8 @@ jobs:
     name: Fuzz
     uses: ./.github/workflows/reusable-fuzz.yml
     with:
-      duration: 600 # seconds == 10 minutes
+      # duration: 600 # seconds == 10 minutes
+      duration: 36000 # seconds == 60 minutes
```

These changes **must** be reverted before this is merged.

```suggestion
      duration: 600 # seconds == 10 minutes
```